src/chainstate/stacks/events.rs

```diff
@@ -61,6 +61,7 @@ pub struct StacksTransactionReceipt {
     pub contract_analysis: Option<ContractAnalysis>,
     pub execution_cost: ExecutionCost,
     pub microblock_header: Option<StacksMicroblockHeader>,
+    pub tx_index: u16,
```

Comment:
This needs to be a `u32` at least.  It's possible that in Stacks 2.1, an epoch can have more than 65536 transactions (right now, the max is 11650).  Let's make the field big enough that it's inconceivable that we'll overflow it.